src/AutoConfirmFuchkaRecepie.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by Mahmuda.Naznin on 10/6/2017.
+ */
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class AutoConfirmFuchkaRecepie {
+
+        public static void main(String[] args){
+            WebDriver driver;
+            System.setProperty("webdriver.gecko.driver", "C:\\Users\\mahmuda.naznin\\Downloads\\geckodriver-v0.19.0-win64\\geckodriver.exe");
+
+            driver=new FirefoxDriver();
```

Comment:
@mnaznin Can you create a new issue to change to Google chrome? This PR does not depend on that, it looks good, we can merge, just for future, lets use Google chrome.